releasenotes/notes/fix-dsd-stale-unix-socket-542e6b1367e45d35.yaml

```diff
@@ -0,0 +1,2 @@
+fixes:
```

Comment:
I think it could be listed under `enhancements` instead of `fixes`:

> the clean-up of the UNIX socket is now done directly by dogstatsd instead of the image entrypoint